nsq/reader.py

```diff
@@ -275,6 +282,15 @@ def set_message_handler(self, message_handler):
         assert callable(message_handler), 'message_handler must be callable'
         self.message_handler = message_handler
 
+    def set_message_timeout_handler(self, message_timeout_handler):
```

Comment:
We only need `set_message_handler` because in certain cases you want to pass your `nsq.Reader` instance to your handler, so it needs to be instantiated before you can be passed in.

Is that why we need this `set_message_timeout_handler`?